src/view/views/RobotView.tsx

```diff
@@ -29,8 +29,9 @@ export const RobotView = () => {
         <Divider />
         <div className="robot-view--stats">
           <StatusTile label="Robot Motors" value={motorStats.length} />
-          {motorStats.map(([label, value]) => (
+          {motorStats.map(([label, value], index) => (
```

Comment:
I saw that [React suggest not to use index as key](https://reactjs.org/docs/lists-and-keys.html#keys) even though I'm pretty sure that in this case it's not really dynamic view.

Do you think we can use the sensor type+channel as unique key?
We can add `type` field to `Sensor` interface.